<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="pricingBoard">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>751</width>
     <height>431</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>No.1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Rank</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>U.Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>L.Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Reduce</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Profit</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="updateTime">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>470</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Oct.01.1990 12:00:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshButton">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>30</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openUrlButton">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>110</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>30</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="detailBrowser">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>160</y>
     <width>181</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="offerButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>110</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Offer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>470</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Updated at: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
